src/Service.Tests/Unittests/RuntimeConfigLoaderJsonDeserializerTests.cs

```diff
@@ -83,13 +83,13 @@ public class RuntimeConfigLoaderJsonDeserializerTests
                 if (replaceEnvVar)
                 {
                     Assert.IsTrue(RuntimeConfigLoader.TryParseConfig(
-                        GetModifiedJsonString(repValues, @"""mssql"""), out expectedConfig, replaceEnvVar: replaceEnvVar),
+                        GetModifiedJsonString(repValues, @"""postgresql"""), out expectedConfig, replaceEnvVar: replaceEnvVar),
                         "Should read the expected config");
                 }
                 else
                 {
                     Assert.IsTrue(RuntimeConfigLoader.TryParseConfig(
-                        GetModifiedJsonString(repKeys, @"""mssql"""), out expectedConfig, replaceEnvVar: replaceEnvVar),
+                        GetModifiedJsonString(repKeys, @"""postgresql"""), out expectedConfig, replaceEnvVar: replaceEnvVar),
```

Comment:
why these changes?

Reply:
this test doesnt use a valid connection string format, so if we use mssql db type we end up inside the code where we will get an exception now during post processing. we want to avoid that for this test, so using postgressql is an easy way to do that.

Reply:
followed up on Teams: Our configloader looks for "MSSQL" dbtype to decide whether to scrutinize the connection string and subject it to the connectionstringbuilder. 

This test is evaluating env_Var replacement which may end up creating an intentionally bad connection string, however, the test is only checking for whether env_vars were processed properly by checking whether configloading succeeded. Now that we have connecitonstringbuilder validation in the runtimeconfigloader, the test broke because of a different untargeted failure.